evm/v0.5/contracts/dev/OracleDev.sol

```diff
@@ -18,4 +18,36 @@ contract OracleDev is Oracle {
   function setPriceFeed(address _priceFeed, bytes32 _currency) external onlyOwner {
     priceFeeds[_currency] = LinkExInterface(_priceFeed);
   }
+
+  //////////////////////////////////////////////////////////////////////////////
+  // Unused concrete methods required by abstract methods on Oracle.sol
```

Comment:
This is weird, these should all get inherited from Oracle.sol. I would actually expect these to break some tests. Is OracleDev.sol not tested?

Reply:
There is an `OracleDev_test.js`, but all tests in it pass, with or without these implementations. You're right, though, not sure what I was thinking, here, since these *are* concrete in `Oracle.sol`. I will delete them.